src/JsonView.tsx

```diff
@@ -7,9 +7,22 @@ const MAX_CHARACTERS = 200;
 const MAX_CHARACTERS_REVEAL_FIRST_N = 100;
 const MAX_CHARACTERS_REVEAL_LAST_N = 100;
 
+const existingKeys: string[] = [];
+
+function incrementTillNew(key: string, suffix: number): string {
```

Comment:
This was added to resolve the react warning about elements having unique key props. It only appears when a document has multiple of the same "leaf" properties (e.g. user.name, issuer.name)

The old logic only set it to "name" which resulted in duplicate keys.
Technically just a v4 uuid could work too.